### PR TITLE
Display price on booking form

### PR DESCRIPTION
**Pull request by ericblandet**
modified the booking form

### CODE REVIEW

File: app/javascript/components/total_price.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ const computePrice = () => {
       });
     });
 
+    // numberOfPersonsInput.addEventListener('change', () => {
+    //   persons.innerText = numberOfPersonsInput.value
+    // });
+
```
Comment (Simon-Busch) [inline app/javascript/components/total_price.js]: Can you delete if not used ? 👍🏼

File: app/javascript/components/total_price.js [hunk 2]
```diff
@@ -3,8 +3,8 @@ const price = document.getElementById("computed_price");
 const startDateInput = document.getElementById("booking_start_date");
 const endDateInput = document.getElementById("booking_end_date");
 const numberOfPersonsInput = document.getElementById("booking_number_of_persons");
-const inputs = document.querySelectorAll(".form-control")
-
+const inputs = document.querySelectorAll(".form-control");
+// const persons = document.getElementById("persons");
```
Comment (Simon-Busch) [inline app/javascript/components/total_price.js]: Can you delete if not used ? 👍🏼

**Review by Simon-Busch — CHANGES_REQUESTED** [review]
Just minor changes

File: app/javascript/components/total_price.js (hunk 1 above)
Comment (ericblandet) [reply app/javascript/components/total_price.js]: yes, done

Comment (Simon-Busch) [reply app/javascript/components/total_price.js]: Parfait

**Pull request merged**